pkg/docker/docker.go

```diff
@@ -528,22 +506,11 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) (err error) {
 	defer d.RemoveContainer(container.ID)
 
 	glog.V(2).Infof("Attaching to container")
-	// creating / piping the channels in runContainerAttachOne lead to unintended hangs
+	// creating / piping the channels in runContainerAttach lead to unintended hangs
 	attached := make(chan struct{})
-	wg := runContainerAttachOne(attached, container, opts, d)
+	wg := runContainerAttach(attached, container, opts, d)
 	attached <- <-attached
```

Comment:
why are we feeding the output of the attached channel back into attached channel?